lib/cylc/gui/app_gcylc.py

```diff
@@ -509,6 +509,29 @@ class ControlApp(object):
         VIEWS["graph"] = ControlGraph
         VIEWS_ORDERED.append("graph")
 
+    STATES_TRIGGERABLE = [
+        'waiting',
+        'held',
+        'queued',
+        'ready',
+        'expired',
+        'submit-failed',
+        'succeeded',
+        'failed',
+        'runahead'
+    ]
+
+    STATES_KILLABLE = [
+        'submitted',
+        'running'
+    ]
+
+    STATES_POLLABLE = [
+        'submitted',
+        'running'
+    ]
```

Comment:
I wonder if we should make 'succeeded' and 'failed' pollable too?  The poll would consult the task status file, and would therefore distinguish between tasks that really finished, and those whose states were manually reset to succeed or failed... which might occasionally be useful.